docs/account_creation.rst

```diff
@@ -0,0 +1,37 @@
+Account creation
+================
+
+Creating an account on StarkNet
+-------------------------------
+
+An account is needed to start interacting with StarkNet.
+If you don't have one there are a few ways of creating one programmatically:
+
+ - using DeployAccount transaction
+ - deploy through Cairo syscall (another account is needed)
+
+.. note::
+
+    In the future users would be able to use
+    `Universal Deployer Contract <https://community.starknet.io/t/universal-deployer-contract-proposal/1864>`_
+    to deploy any contract, including an Account.
```

Comment:
```suggestion
    to deploy contracts.
```

Deploying Account will be possible, but if you do not posses any account currently, using UDC won't be useful at all.